lib/modules/manager/unity3d/index.ts

```diff
@@ -0,0 +1,18 @@
+import type { Category } from '../../../constants';
+import { Unity3dDatasource } from '../../datasource/unity3d';
+import { fileMatchRegex } from './extract';
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: fileMatchRegex,
+  packageRules: [
+    {
+      groupName: 'Unity Editor',
+      matchPackageNames: ['m_EditorVersion', 'm_EditorVersionWithRevision'],
+    },
+  ],
```

Comment:
This should be rather part of a preset, as it is hard to find out what is happening if the config is set here. 

If we are talking about the same version in different places you can set `depName` to the same value. This will by default result too in a single PR